### ISSUE TITLE
README.md does not get you started

### ISSUE THREAD

**Issue by ajbraus** [auth0/express-jwt#156]
Hi there, the README is very hard to parse out exactly how to use express-jwt. Since this is a relatively new technology, why not simplify exactly how someone can login/register using it? Maybe a sample app? 

If I get things sorted out myself I'll circle back around and do a PR, but no promises. 

Thanks!

**Comment by chriswhong** [comment]
+1, I get how to implement the package on the server, but there are no examples of how the jwt should be attached to the incoming request.  (I am using `Authorization: Bearer {jwt}` header but am getting a `jwt malformed` error.  An example of a valid request would be super helpful.

**Comment by gonrial** [comment]
A complete example, including the class `data` will help to understand all the scopes

**Comment by psotos** [comment]
+1

has anyone produced a complete working example yet? If not, would you guys like to work with me to create one?

**Comment by SuperTango** [comment]
+1.  How does one actually create the JWT when a login occurs?  What should the subject be? how is it translated back to a req.user?  So many questions.  :)

@psotos I'd be happy to help, but I don't have it working yet.

**Comment by willxy** [comment]
As best I can tell, it looks like `express-jwt` is just for validating incoming jwt's ("validates JsonWebTokens and sets req.user"), and that other parts of the process, like creating jwt's, fall to other modules like [`jsonwebtoken`](https://github.com/auth0/node-jsonwebtoken) or [`jwt-express`](https://github.com/AustP/jwt-express).

**Comment by iamdubx** [comment]
Any news? I am struggling with switch from passport to this Auth0 supported library, but seems like this guys are busy at counting their $30M? than making simple example with own library to newbies!
